serving/src/main/java/ai/djl/serving/http/InferenceRequestHandler.java

```diff
@@ -64,36 +63,32 @@ protected void handleRequest(
             throws ModelException {
         switch (segments[1]) {
             case "ping":
-                ModelManager.getInstance().workerStatus(ctx);
+                // TODO: Check if its OK to send other 2xx errors to ALB for "Partial Healthy"
+                // and "Unhealthy"
+                ModelManager.getInstance()
+                        .workerStatus(ctx)
+                        .thenAccept(
+                                response ->
+                                        NettyUtils.sendJsonResponse(
+                                                ctx,
+                                                new StatusResponse(response),
+                                                HttpResponseStatus.OK));
                 break;
             case "invocations":
                 handleInvocations(ctx, req, decoder);
                 break;
-            case "predictions":
```

Comment:
have a look into InterferencePredictionRequestHandler. I moved the code to a separate Handler-class, I would suggest to do the same for invocations and ping. Would be a cleanCode/OODesign refactoring to avoid switch-case by using classes instead.  cleaner short classes with a single responsibility per class and might be easier to find that there is a PING. I first have overseen this till I digged deep in to classes.